files/issue.txt

```diff
@@ -14,5 +14,5 @@
      #####         ::::::::    :        
       ######        ####:::::::::       https://docs.photonvision.org/
        ##################    ::::::     
-         ##############                 
-           #########                    
+         ##############                 The SSH user:password pair for
```

Comment:
Can we word this more plainly? I wonder if "the username is photon and the password is vision" is more clear